TypeCobol.LanguageServer/Workspace.cs

```diff
@@ -125,6 +128,30 @@ public bool TimerDisabledOption
         /// </summary>
         public bool UseSyntaxColoring { get; set; }
 
+#if EUROINFO_RULES
+        /// <summary>
+        /// The Cpy Copy names file
+        /// </summary>
+        public string CpyCopyNamesMapFilePath
+        {
+            get
+            {
+                return this.CompilationProject.CompilationOptions.CpyCopyNamesMapFilePath;
+            }
+            set
+            {
+                try
+                {
+                    this.CompilationProject.CompilationOptions.CpyCopyNamesMapFilePath = value;
+                }
+                catch (Exception e)
+                {
+                    this._Logger(TypeCobolConfiguration.ErrorMessages[ReturnCode.CopyNameMapFileError] + ":" + e.Message, null);
+                }
+            }
+        }
```

Comment:
Ok this is to ensure that we have the correct file path at initialization time but this is specific to this particular option. Other options are not considered at initialization time, they are in the compilation project only on first `DidChangeConfiguration` notification. I'm not sure we should make an exception here. See my remark below.

Reply:
The copy name file is a very special option, the difficulty here, is that sometime we consider it as a global setting, and sometime we consider it as a specific setting.

Reply:
I Could not change this now, because this options will not be set using DidChangeConfiguration, but only using sever command line options. So it is really a specific option.